server/graphql/v2/mutation/AccountMutations.ts

```diff
@@ -941,6 +941,65 @@ const accountMutations = {
       return recoveryCodesArray;
     },
   },
+  convertAccountToOrganization: {
+    type: new GraphQLNonNull(GraphQLAccount),
+    description: 'Convert an account to an Organization. Scope: "account".',
+    args: {
+      account: {
+        type: new GraphQLNonNull(GraphQLAccountReferenceInput),
+        description: 'Account to convert.',
+      },
+      hasMoneyManagement: {
+        type: GraphQLBoolean,
+        defaultValue: false,
+        description: 'Should the Organization have money management capabilities',
+      },
+      legalName: {
+        type: GraphQLString,
+        description: 'Legal name of the Organization.',
+      },
+    },
+    async resolve(_: void, args, req: express.Request): Promise<Collective> {
+      checkRemoteUserCanUseAccount(req);
+
+      const account = await fetchAccountWithReference(args.account, { loaders: req.loaders, throwIfMissing: true });
+
+      if (!req.remoteUser.isAdminOfCollective(account) && !req.remoteUser.isRoot()) {
+        throw new Forbidden();
+      }
+
+      // Disallow if Hosted
+      if (account.HostCollectiveId && account.approvedAt) {
+        throw new Error("Can't convert an hosted Collective.");
+      }
+
+      await TwoFactorAuthLib.enforceForAccount(req, account, { alwaysAskForToken: true });
+
+      await account.update({ type: CollectiveType.ORGANIZATION });
+
+      if (args.legalName) {
+        await account.update({ legalName: args.legalName });
+      }
```

Comment:
Unless we want to make that a mutation parameter 

```suggestion
      await account.update({ 
        type: CollectiveType.ORGANIZATION,
        legalName: args.legalName || account.legalName,
        data: { ...account.data, canHostAccounts: false },
      });
```